css/ryht-charter-campuses.css

```diff
@@ -534,6 +535,25 @@ p.moreinfo  {
     display: none;
 }
 
+input.autocompleteTextbox, select.dropdown {
+	display: block;
+	margin-top: 1ex;
+	margin-bottom: 1ex;
+}
```

Comment:
This bit is just replacing the `<br>` tags I took out of index.html with something more configurable